adapter/internal/adapter/adapter.go

```diff
@@ -223,8 +224,23 @@ func Run(conf *config.Config) {
 	if eventHubEnabled {
 		// Load subscription data
 		eventhub.LoadSubscriptionData(conf)
+		var isAzureEventingFeatureFlagEnabled bool
+		var err error
+
+		envValue := os.Getenv("FEAURE_FLAG_REPLACE_EVENT_HUB")
+		if (envValue != "") {
+			isAzureEventingFeatureFlagEnabled, err = strconv.ParseBool(envValue)
+			if (err != nil) {
+				logger.LoggerMgw.Error("Error occurred while parsing FEAURE_FLAG_REPLACE_EVENT_HUB environment value.",
```

Comment:
shall we make this an info log in the form we discussed? Because we don't need to hi any alerting systems as this is a test feature.

Reply:
sure